pyproject.toml

```diff
@@ -13,7 +13,10 @@ classifiers = [
 ]
 requires-python = ">=3.8"
 dependencies = [
-  "setuptools_scm[toml]>=6.2"
+  "setuptools_scm[toml]>=6.2",
+  "lxml",
+  "numpy",
+  "typing_extensions",
```

Comment:
TODO: add versions

Reply:
I was thinking of minimum requirements here, we probably don't need to limit the versions more than that. I believe both `lxml` and `typing_extensions` are already pulled in by Zephyr/NCS, so I will try to make sure that the versions are compatible with those already required.